packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -53,6 +53,53 @@ function createTest(name, file, it) {
   });
 }
 
+/**
+ * Run a test by merging a base template with properties from another file.
+ * @param {string} name - name of the base template
+ * @param {string} additionalPropertiesName - Name of the file containing additional properties
+ * @param {Mocha.TestFunction} it - Mocha's `it` function to run the test
+ */
+function createTestWithProperties(name, additionalPropertiesName, bindingType, it) {
```

Comment:
This might be a little over-engineering, at the same time, I also did not feel writing 30 fixture files was a great idea.